source/blog/2016-05-19-introducing-subteams.md

```diff
@@ -0,0 +1,148 @@
+  for, major initiatives in the project.
+* **Setting priorities and the release schedule.** Design bandwidth is limited,
+  and it's dangerous to try to grow the framework too quickly; the core
+  team makes some difficult decisions about which areas to prioritize for
+  new design, based on the core values and target use cases.
+  The regular release cycle, being one of the project's core values,
+  applies to subteams as well.
+* **Focusing on broad, cross-cutting concerns.** The core team is specifically
+  designed to take a global view of the project, to make sure the pieces
+  are fitting together in a coherent way.
+* **Spinning up or shuting down subteams.** Over time, we may want to expand the
+  set of subteams, and it may make sense to have temporary "strike
+  teams" that focus on a particular, limited task.
+* **Going/no-going features**. While the subteams make decisions on RFCs,
+  the core team is responsible for pulling the trigger that moves a
+  feature from canary to beta. This provides an extra check that
+  features have adequately addressed cross-cutting concerns and that the
+  implementation quality is high enough
+
+Each subteam is lead by a core team liaison. This core team member
+is responsible for keeping the core team apprised of progress on
+important initiatives, identifying potential areas of concern early, and
+escalating cross-cutting issues when needed.
+
+## Ember CLI
+
+**Slack channel**: #dev-ember-cli  
+**Discourse category**: [Ember CLI](http://discuss.emberjs.com/c/ember-cli)
+
+**Core Team Liaisons**: Stefan Penner & Robert Jackson  
+
+**Members**:
+
+* Katie Gengler
+* Chad Hietala
+* Jake Bixby
+* Nathan Hammond
+* Kelly Selden
+* Tobias Bieniek
+
+## Ember Data
+
+**Slack channel**: #dev-ember-data  
+**Discourse category**: [Ember
+Data](http://discuss.emberjs.com/c/ember-data)  
+
+**Core Team Liaison**: Igor Terzic  
+
+**Members**:
+
+* Clemens Müller
+* Christoffer Persson
+* Stanley Stuart
+* Brendan McLoughlin
+
+## Learning
+
+**Slack channel**: #learning  
+**Discourse category**: [Learning](http://discuss.emberjs.com/c/learning)
+
+While the Ember CLI and Ember Data subteams should be self-explanatory,
+this one is the newest and may require some explanation.
+
+We intentionally did not call this subteam _Documentation_,
+because documentation is just one part of how new users learn to use
+Ember. People start learning the second they land on the website
+homepage, or when they see a presentation at their local user group.
+
+We want to make holistic learning a central part of Ember. In addition to good
+guides and API documentation, this means thinking about how members of
+the community interact, how they get help, and how we introduce them to
+new features added via the RFC process.
+
+The Learning Subteam is responsible for the website, the guides, the API
+docs and making sure the core team is aware of common pitfalls people
```

Comment:
Unsure about the correct capitalization of "core team", but lower case looks odd if "Learning Subteam" is capitalized.